deployment/grid/charts/values/keda.yaml

```diff
@@ -14,32 +15,27 @@ podSecurityContext:
 
 image:
   keda:
-    repository: ${aws_htc_ecr}/kedacore/keda
-    # Allows people to override tag if they don't want to use the app version
-    tag: ${k8s_keda_version}
+    repository: "${aws_htc_ecr}/kedacore/keda"
```

Comment:
Is there a reason for tag to be removed from helm configuration

Reply:
The reason for this change is coz I think it would be a good idea to treat the Helm releases same as other modules and control them via the Helm release version rather than fiddling with the individual component tags. This could also lead to cases where the Helm release is updated but the images are kept back to older versions due to someone not updating the version or not knowing about it, etc. This way we have a unform control over the release with a single variable and everything else follows suit. (I am aware that in the case of Keda, the version of thre release and the images does match and it is the same variable, but its not the case for other modules/components, so I think that unifying the way we handle those will be valuable for the future).